libs/barista-components/event-chart/src/event-chart.ts

```diff
@@ -460,15 +589,106 @@ export class DtEventChart<T> implements AfterContentInit, OnInit, OnDestroy {
     return path.join(' ');
   }
 
+  _calculateFieldOutline(
+    renderEvent: RenderField<T>,
+    offset: number = 0,
+  ): string {
+    const eventBubbleRadius = FIELD_BUBBLE_SIZE + offset;
```

Comment:
You are creating a rectangle with rounded corners, if I'm not mistaken here. 
Any reason why you are not using `rx` on a rectangle to add the rounded corners like this:
`<rect x="120" width="100" height="100" rx="15" />`

Reply:
I've had some strange problems with the `rect`